### DISCUSSION TITLE
OIDC callback with cli proxy + ory cloud

### DISCUSSION THREAD

**Discussion by Rodeoclash** [ory/kratos#2772] [Q&A]
I might have missed a configuration option somewhere, but when performing an OIDC flow (in this case, a custom OIDC provider which is Xero) the callback path to the browser seems to be missing the `.ory` prefix.

## Flow

![1](https://user-images.githubusercontent.com/63807/193378987-edc939df-ae86-4156-a38e-c49184b20dde.png)
**Starting login flow**

![2](https://user-images.githubusercontent.com/63807/193378998-254e6c97-2e9c-4edb-9d41-a81bf88f6896.png)
**Selecting Xero**

![3](https://user-images.githubusercontent.com/63807/193379004-bd4e2cfc-821b-4378-b0dc-7653e5e5953e.png)
**Auth success, but notice the URL is missing the `.ory` prefix**

![4](https://user-images.githubusercontent.com/63807/193379025-f6676607-811d-4efe-82a7-dd7708fc7345.png)
**Manually updated the URL, everything works**

Is this an edge case of local development, the ory proxy and social sign in? I can fix by setting up an automatic redirect in my code but was curious to know if I had misconfigured something.

**Comment by aeneasr** [comment]
That’s quite possible! We’ll investigate next week!

**Comment by Rodeoclash** [comment]
For what it's worth, this probably me not understanding or incorrectly setting one of the return url fields in the Kratos config.

I've since moved to developing against a local copy of Kratos which is working much better. I think this can be closed and chalked up to developer misunderstanding.

**Reply by aeneasr** [reply]
@Rodeoclash there was actually a bug in the proxy which I have resolved. We'll release it today :)
